workflows/src/test/kotlin/com/template/FlowTests.kt

```diff
@@ -1,14 +1,29 @@
 package com.template
 
 import net.corda.testing.node.MockNetwork
+import net.corda.testing.node.MockNetworkParameters
+import net.corda.testing.node.TestCordapp
 import org.junit.After
 import org.junit.Before
 import org.junit.Test
 
 class FlowTests {
-    private val network = MockNetwork(listOf("com.template"))
-    private val a = network.createNode()
-    private val b = network.createNode()
+    private val network = MockNetwork(
+        MockNetworkParameters()
+            .withCordappsForAllNodes(
+                listOf(
+                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.contracts"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.workflows"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.contracts"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.workflows"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.money"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.selection"),
```

Comment:
I believe you don't need to explicitly list `selection` here:

See the JIT MockNetwork tests found here: 

```TestCordapp.findCordapp("com.r3.corda.lib.tokens.money"),
                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.contracts"),
                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.workflows")```

Reply:
It looks like selection was also moved to `workflows` in `1.2`

https://docs.corda.net/docs/corda-os/4.5/token-sdk-introduction.html#upgrade-from-v-11-to-12

Reply:
Fixed